### ISSUE TITLE
export onnx model failed

### ISSUE THREAD

**Issue by UNeedCryDear** [ultralytics/yolov5#6643]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar bug report.

### YOLOv5 Component

Export

### Bug

The latest code export model failed, and the previous code（about ten days ago） can be exported correctly.

### Environment

- yolov5：Latest
- windows 10
- torch 1.8.2+cu102

### Minimal Reproducible Example

The codes used are official, but the time period is different.

### Additional

previous export onnx model ok 
![55](https://user-images.githubusercontent.com/52729998/154000756-e68b7996-76c5-4849-ae6d-f600d7e6d8b0.png)

lasted：
![11](https://user-images.githubusercontent.com/52729998/154001049-f1d8ab6f-b201-4d5d-b9e2-05f50c83b107.png)

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@UNeedCryDear 👋 hi, thanks for letting us know about this possible problem with YOLOv5 🚀. I just verified ONNX export now using master in Colab and everything works correctly:

<img width="1595" alt="Screenshot 2022-02-15 at 10 58 43" src="https://user-images.githubusercontent.com/26833433/154038459-39c0b6b9-49c7-4890-a953-690c191378f1.png">

We've created a few short guidelines below to help users provide what we need in order to get started investigating a possible problem.

## How to create a Minimal, Reproducible Example

When asking a question, people will be better able to provide help if you provide **code** that they can easily understand and use to **reproduce** the problem. This is referred to by community members as creating a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/). Your code that reproduces the problem should be:

- ✅ **Minimal** – Use as little code as possible to produce the problem
- ✅ **Complete** – Provide all parts someone else needs to reproduce the problem
- ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem

For [Ultralytics](https://ultralytics.com/) to provide assistance your code should also be:

- ✅ **Current** – Verify that your code is up-to-date with GitHub [master](https://github.com/ultralytics/yolov5/tree/master), and if necessary `git pull` or `git clone` a new copy to ensure your problem has not already been solved in [master](https://github.com/ultralytics/yolov5/tree/master).
- ✅ **Unmodified** – Your problem must be reproducible using official YOLOv5 code without changes. [Ultralytics](https://ultralytics.com/) does not provide support for custom code ⚠️.

If you believe your problem meets all the above criteria, please close this issue and raise a new one using the 🐛 **Bug Report** [template](https://github.com/ultralytics/yolov5/issues/new/choose) with a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us better understand and diagnose your problem. 

Thank you! 😃

**Comment by UNeedCryDear** [comment]
I think I have just found the crux of the problem.
When I renmove pandas and the function export_formats which is defined but not used，export.py  works correctly.
So I wonder if it's my pandas version doesn't match. my pandas version is 1.3.4>=1.1.4 which in requirements.txt.
When I update pandas to the latest(1.4.1), it works normally.So is pandas redundant here?

**Comment by glenn-jocher** [comment]
@UNeedCryDear ah thanks for the info! Colab also works fine with 1.3.5 installed by default. Can you try that version and let me know if export works for you?

the export_formats function is used by utils/benchmarks.py and will be leaned on for future export.py updates so we need to retain the pandas import.

**Comment by glenn-jocher** [comment]
@UNeedCryDear I installed and exported successfully with pandas==1.3.4 in Colab. So I'm unable to reproduce a pandas problem.

<img width="1336" alt="Screenshot 2022-02-16 at 15 17 40" src="https://user-images.githubusercontent.com/26833433/154283295-2cbc9a2e-7a1e-4a93-8378-11c6aa889591.png">

**Comment by glenn-jocher** [comment]
@UNeedCryDear also successfully export with pandas==1.1.4 in Colab. I think then no action is required here, removing TODO.

<img width="1336" alt="Screenshot 2022-02-16 at 15 19 25" src="https://user-images.githubusercontent.com/26833433/154283627-531497ac-84f4-4102-8202-5bf82ee0824e.png">

**Comment by UNeedCryDear** [comment]
Unfortunately, even if I change a computer and the new virtual environment only installs the requirements in the requirements.txt, it still can't work.
I have tried all versions of pandas, and only versions>=1.4.0 will work.
this is a new virtual environment list:
![19](https://user-images.githubusercontent.com/52729998/154301481-901f8d79-e90c-4174-a1d8-39e6ce6c30a8.png)
new environment：
OS: windows 10
CPU/GPU：AMD 5800X/ GTX 3060
CUDA/CUDNN: CU11.1.1 +CUDNN8.2.1

**Comment by UNeedCryDear** [comment]
I changed the third computer to test, which is the same problem.
At the same time, I run it on Ubuntu and it can be exported correctly with pandas>=1.1.4. It looks like a problem with windows 10. The previous three computers all had such problems on winsows 10

**Comment by phodgers** [comment]
Hello, I am here since I want to try and get Intel Neural Compute Stick 2 working. I tried the export with -openvino option. I can confirm I saw the same issue on Windows10 - it seems to fall over when attempting the ONNX export. I uninstalled pandas and re-installed and it automatically installed panda v 1.4.1 which now seems to enable the export to complete. However it does then give some error message about invalid requirement 'openvino-dev'.

> C:\Users\Philip\Downloads\yolov5>pip install pandas
> Collecting pandas
>   Downloading pandas-1.4.1-cp39-cp39-win_amd64.whl (10.5 MB)
>      ---------------------------------------- 10.5/10.5 MB 16.8 MB/s eta 0:00:00
> Requirement already satisfied: numpy>=1.18.5 in c:\users\philip\appdata\roaming\python\python39\site-packages (from pandas) (1.19.5)
> Requirement already satisfied: python-dateutil>=2.8.1 in c:\users\philip\appdata\local\programs\python\python39\lib\site-packages (from pandas) (2.8.2)
> Requirement already satisfied: pytz>=2020.1 in c:\users\philip\appdata\local\programs\python\python39\lib\site-packages (from pandas) (2021.3)
> Requirement already satisfied: six>=1.5 in c:\users\philip\appdata\local\programs\python\python39\lib\site-packages (from python-dateutil>=2.8.1->pandas) (1.16.0)
> Installing collected packages: pandas
> ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
> openvino-dev 2021.4.2 requires pandas~=1.1.5, but you have pandas 1.4.1 which is incompatible.
> Successfully installed pandas-1.4.1
> 
> C:\Users\Philip\Downloads\yolov5>python export.py --include openvino
> export: data=data\coco128.yaml, weights=yolov5s.pt, imgsz=[640, 640], batch_size=1, device=cpu, half=False, inplace=False, train=False, optimize=False, int8=False, dynamic=False, simplify=False, opset=12, verbose=False, workspace=4, nms=False, agnostic_nms=False, topk_per_class=100, topk_all=100, iou_thres=0.45, conf_thres=0.25, include=['openvino']
> YOLOv5  v6.0-265-g18b5081 torch 1.10.0+cu113 CPU
> 
> Fusing layers...
> Model Summary: 213 layers, 7225885 parameters, 0 gradients
> 
> PyTorch: starting from yolov5s.pt with output shape (1, 25200, 85) (14.7 MB)
> 
> ONNX: starting export with onnx 1.10.2...
> ONNX: export success, saved as yolov5s.onnx (29.3 MB)
> requirements: openvino-dev not found and is required by YOLOv5, attempting auto-update...
> ERROR: Invalid requirement: "'openvino-dev'"
> requirements: Command 'pip install 'openvino-dev'' returned non-zero exit status 1.
> 
> OpenVINO: starting export with openvino 2021.4.2-3976-0943ed67223-refs/pull/539/head...
> OpenVINO: export success, saved as yolov5s_openvino_model\ (29.6 MB)
> 
> Export complete (17.39s)
> Results saved to C:\Users\Philip\Downloads\yolov5
> Detect:          python detect.py --weights yolov5s_openvino_model\
> PyTorch Hub:     model = torch.hub.load('ultralytics/yolov5', 'custom', 'yolov5s_openvino_model\')
> Validate:        python val.py --weights yolov5s_openvino_model\
> Visualize:       https://netron.app
> 
> C:\Users\Philip\Downloads\yolov5>

**Comment by glenn-jocher** [comment]
@phodgers @UNeedCryDear yeah, we are stuck between these two. It seems windows requires `pandas>=1.4.0` but openvino-dev (required for OpenVINO export) required pinned `pandas==1.1.5`. 

This is all extra strange since pandas doesn't actually do anything at the moment in export.py except get imported. We were planning on using it later though since the dataframes with column names print very nicely.
